README.md

```diff
@@ -114,9 +122,11 @@ journalctl -fu docker.service # Journalctl/Systemd
 
 **VM (Docker-side) Plugin logs**
 
-* Log location: `/var/log/docker-volume-vsphere.log`
-* Config file location: `/etc/docker-volume-vsphere.conf`.
- * This JSON-formatted file controls logs retention, size for rotation
+* Log location (Linux): `/var/log/docker-volume-vsphere.log`
+* Log location (Windows): `C:\Windows\System32\config\systemprofile\AppData\Local\docker-volume-vsphere\logs\docker-volume-vsphere.log`
```

Comment:
Yes, we can file a separate PR to track the improvement of moving logs to C:\ProgramData\VMware\vDVS\Logs\. But can you please verify above location (C:\Windows\System32\config\...) is correct for both Windows 10 and Windows Server 2016, which we have announced official support in the document below?